src/Component/Calculator.js

```diff
@@ -1,75 +1,76 @@
-import React from 'react';
+/* eslint-disable */
+import React, { useState } from 'react';
+/* eslint-enable */
 import calculate from '../logic/calculate';
+import updateObject from '../logic/update';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.eventHandler = this.eventHandler.bind(this);
-    this.state = {
-      total: '',
-      next: '',
-      operation: '',
-    };
-  }
+const Calculator = () => {
+  const [stotal, setTotal] = useState('');
+  const [soperation, setOperation] = useState('');
+  const [snext, setNext] = useState('');
 
-  eventHandler(newState) {
-    if (newState.total != null) {
-      this.setState({ total: newState.total });
-    }
+  const obj = {
+    total: stotal,
+    next: snext,
+    operation: soperation,
+  };
+  const eventHandler = (newState) => {
+    // if (newState.total != null) {
+    //   setTotal(newState.total);
+    // }
```

Comment:
- [ ] Kindly remove any commented-out code from your files. This is best practices for cleaner and more optimized code 😄 🙏